requirements.txt

```diff
@@ -1 +1,7 @@
 nose==1.3.4
+Fiona==1.5.1
+numpy==1.10.1
+pandas==0.17.1
+pyshp==1.2.3
+scipy==0.16.1
+Shapely==1.2.19
```

Comment:
We may be able to target the latest versions of these libraries since GDAL and Proj4 are built from source now.